setup.py

```diff
@@ -59,6 +59,10 @@
 It integrates with many standard python libraries to give you out-of-the-box
 logging, metrics, error reporting, status urls and more.
 
+Python version support
+----------------------
+
+This release of talisker (0.20.0) will be the last to support python 2.7
```

Comment:
Oh I see, this looked a bit confusing at first but I see the README file contains this warning already.